src/manifolds/MultinomialSymmetricPositiveDefinite.jl

```diff
@@ -0,0 +1,52 @@
+@doc raw"""
+    MultinomialSymmetricPositiveDefinite <: AbstractMultinomialDoublyStochastic
+
+The symmetric positive definite multinomial matrices manifold consists of all
+symmetric ``n×n`` matrices with positive eigenvalues, and
+positive entries such that each column sums to one, i.e.
+
+````math
+\begin{aligned}
+\mathcal{SP}^+(n) \coloneqq \bigl\{
+    p ∈ ℝ^{n×n}\ \big|\ &p_{i,j} > 0 \text{ for all } i=1,…,n, j=1,…,m,\\
+& p^\mathrm{T} = p,\\
+& p\mathbf{1}_n = \mathbf{1}_n\\
+a^\mathrm{T}pa > 0 \text{ for all } a ∈ ℝ^{n}\backslash\{\mathbf{0}_n\}
+\bigr\},
+\end{aligned}
+````
+
+where ``\mathbf{1}_n`` and ``\mathbr{0}_n`` are the vectors of length ``n``
+containing ones and zeros, respectively. More details for this manifold can b found in
```

Comment:
```suggestion
containing ones and zeros, respectively. More details for about manifold can be found in
```

Reply:
`about` sounds better here, but then without the `for` upfront.

Reply:
Yes, I actually meant to replace "for", not "this".